emails/views.py

```diff
@@ -343,11 +343,13 @@ def _get_text_and_html_content(email_message):
     if email_message.is_multipart():
         for part in email_message.walk():
             try:
-                content = part.get_content()
+                # multipart/* are just containers
+                if part.get_content_maintype() == 'multipart':
+                    continue
```

Comment:
I would remove this for now since the `multipart/*` would just get ignored in the `if` statements. I am also worried about the multipart having the attachment which will not get checked with this `continue`
```suggestion
```

Reply:
Oh good point!